src/content/components/Router/Router.js

```diff
@@ -116,9 +117,7 @@ export class Router extends React.PureComponent {
       })
       .map(meta => ({
         path: `/feature/${meta.id}`,
-        label: `${meta.priority !== "P1" ? `[${meta.priority}] ` : ""}${
```

Comment:
This wasn't necessary because the filter above at line 102 makes it impossible for the priority to not be P1. We only show unresolved P1 bugs that block the bugzy epic.